app/room_request/views.py

```diff
@@ -103,44 +103,49 @@ def delete_room_request(room_request_id):
     return redirect('/room-request/')
 
 
-@room_request.route('/<int:form_id>', methods=['GET', 'POST'])
-def viewID(form_id):
-    form = ActivityForm()
-    trasnferf = TransferForm()
-    user_found = False
```

Comment:
Instead of keeping track of `user_found`, I decided to just return a 404 if the id in the URL doesn't correspond to a room request - I think this is slightly cleaner, and the end user probably won't be accessing a bad id route anyway.